hack/verify-golint.sh

```diff
@@ -25,6 +25,6 @@ cd "${KUBE_ROOT}"
 
 # See configuration file in ${KUBE_ROOT}/.golangci.yml.
 
-docker run --rm -v $(pwd):/app -w /app "golangci/golangci-lint:$VERSION" golangci-lint run
+docker run --rm -v $(pwd):/app -w /app "golangci/golangci-lint:$VERSION" golangci-lint run "$@"
```

Comment:
This change feels a bit out of place in this PR - mind creating a separate one just for this?

Reply:
yeh we can drop this

Reply:
dropped